test/PlaidClientTest.js

```diff
@@ -643,11 +648,7 @@ describe('plaid.Client', () => {
           expect(response).to.be.ok();
           expect(response.report).to.be.ok();
 
-          // The transactions in an Asset Report with Insights should have a
```

Comment:
why is this change necessary?

Reply:
i think the accounts returned are non-deterministic 
it's hard for my to debug this further without having the credentials CI has.